test/regressions/index.js

```diff
@@ -56,31 +56,50 @@ function excludeTest(suite, name) {
   });
 }
 
+const tests = [];
+
 // Also use some of the demos to avoid code duplication.
 const requireDocs = require.context('docsx/data', true, /\.js$/);
-const tests = requireDocs.keys().reduce((res, path) => {
+requireDocs.keys().forEach((path) => {
   const [name, ...suiteArray] = path.replace('./', '').replace('.js', '').split('/').reverse();
   const suite = `docs-${suiteArray.reverse().join('-')}`;
 
   if (excludeTest(suite, name)) {
-    return res;
+    return;
   }
 
   // TODO: Why does webpack include a key for the absolute and relative path?
   // We just want the relative path
   if (!path.startsWith('./')) {
-    return res;
+    return;
   }
 
-  res.push({
+  tests.push({
     path,
     suite,
     name,
     case: requireDocs(path).default,
   });
+});
+
+const requireRegressions = require.context('./data-grid', true, /\.js$/);
```

Comment:
Nit: Probably the naming could include the keyword `Grid`, e.g. `requireGridRegressions` since they are specific to the Grid.